ehrapy/preprocessing/_bias.py

```diff
@@ -0,0 +1,244 @@
+
+
+@check_feature_types
+def detect_bias(
+    adata: AnnData,
+    sensitive_features: Iterable[str] | Literal["all"],
+    *,
+    run_feature_importances: bool | None = None,
+    corr_threshold: float = 0.5,
+    smd_threshold: float = 0.5,
+    categorical_factor_threshold: float = 2,
+    feature_importance_threshold: float = 0.1,
+    prediction_confidence_threshold: float = 0.5,
+    corr_method: Literal["pearson", "spearman"] = "spearman",
+    copy: bool = False,
+) -> dict[str, pd.DataFrame] | tuple[dict[str, pd.DataFrame], AnnData]:
+    """Detects biases in the data using feature correlations, standardized mean differences, and feature importances.
+
+    Detects biases with respect to sensitive features, which can be either a specified subset of features or all features in adata.var.
+    The method detects biases by computing:
+    - pairwise correlations between features
+    - standardized mean differences for numeric features between groups of sensitive features
+    - value counts of categorical features between groups of sensitive features
+    - feature importances for predicting one feature with another.
+    Results of the computations are stored in var, varp, and uns of the adata object.
+    Values that exceed the specified thresholds are considered of interest and returned in the results dictionary.
+
+    Args:
+        adata: An annotated data matrix containing EHR data.
+        sensitive_features: Sensitive features to consider for bias detection. If set to "all", all features in adata.var will be considered.
+            If only a subset of features should be considered, provide as an iterable.
+        run_feature_importances: Whether to run feature importances for detecting bias. If set to None, the function will run feature importances if
+            sensitive_features is not set to "all", as this can be computationally expensive. Defaults to None.
+        corr_threshold: The threshold for the correlation coefficient between two features to be considered of interest. Defaults to 0.5.
+        smd_threshold: The threshold for the standardized mean difference between two features to be considered of interest. Defaults to 0.5.
+        categorical_factor_threshold: The threshold for the factor between the value counts (as percentages) of a feature compared between two
+            groups of a sensitive feature. Defaults to 2.
+        feature_importance_threshold: The threshold for the feature importance of a sensitive feature for predicting another feature to be considered
+            of interest. Defaults to 0.1.
+        prediction_confidence_threshold: The threshold for the prediction confidence (R2 or accuracy) of a sensitive feature for predicting another
+            feature to be considered of interest. Defaults to 0.5.
+        corr_method: The correlation method to use. Choose between "pearson" and "spearman". Defaults to "spearman".
```

Comment:
```suggestion
        corr_method: The correlation method to use. Defaults to "spearman".
```
I double checked it, the Literal type hints already say that